README.md

```diff
@@ -6,7 +6,7 @@ effort of installing it on various systems.
 
 The image is based on Debian Bullseye (slim) and includes:
 
-- Ansible 7.3.0
+- Ansible >= 7.7, &lt; 8.0
```

Comment:
(Observation) It seems odd to allow a literal `>`, but have a `&lt;` on the same line.

Reply:
`<` opens an HTML tag, at least in the syntax highlighting. I suspect that the rendering would look OK.

I would write this as `~> 7.7`, but that's not a Pythonic way of writing these.

I could write this as `Ansible 7 at 7.7.0 or later`.